src/routes/info/[slug]/edit/+page.server.ts

```diff
@@ -0,0 +1,54 @@
+import { Prisma } from "@prisma/client";
+import { error, fail, redirect } from "@sveltejs/kit";
+import prisma from "$lib/utils/prisma";
+import { policyAccessGuard, withAccess } from "$lib/utils/access";
+import type { PageServerLoad } from "./$types";
+
+export const load: PageServerLoad = async ({ parent, params }) => {
+  const markdownPage = await prisma.markdown.findUnique({
+    where: {
+      name: params.slug,
+    },
+  });
+  if (markdownPage == undefined) {
+    throw error(404, {
+      message: "Not found",
+    });
+  }
+  const { accessPolicies } = await parent();
+  policyAccessGuard(`markdowns:${params.slug}:update`, accessPolicies);
+  return { markdown: markdownPage, slug: params.slug };
+};
+
+export const actions = {
+  default: async ({ request, locals }) => {
+    // read the form data sent by the browser
+    const formData = await request.formData();
+    const name = String(formData.get("name"));
+    const session = await locals.getSession();
+    return withAccess(`markdowns:${name}:update`, session?.user, async () => {
```

Comment:
same as before.

Then I personally prefer ctxAccessGuard over withAccess to minimze indentation

Reply:
fixed, kept withAccess since that seems to be a pattern in the rest of the actions